api/lql_execute.go

```diff
@@ -52,15 +59,15 @@ func validateQueryArguments(args []ExecuteQueryArgument) (err error) {
 	var start, end time.Time
 
 	for _, arg := range args {
-		if arg.Name == "StartTimeRange" {
+		if arg.Name == QueryStartTimeRange {
 			hasStart = true
 			start, err = validateQueryTimeString(arg.Value)
 		}
 		if err != nil {
 			return errors.Wrap(err, "invalid StartTimeRange argument")
```

Comment:
You are opening a new issue, this function has named returns `(err error)` but you override all `return XYZ` statements which is an anti-pattern and we should avoid. 

The best thing to do here is to remove the named return and initialize it at the beginning of the function.

Reply:
Opening a new issue?  I have no problem fixing this, but what you're referring to has been in place for 10 mos.

Reply:
Fixed.